internal/service/systemd.go

```diff
@@ -20,12 +20,20 @@ const (
 	DefaultRestartTimeout = 15
 	TimerSuffix           = ".timer"
 	ServiceSuffix         = ".service"
+	DefaultNameSeparator  = "-"
```

Comment:
this isn't used.

Reply:
Deleted. I wonder why no checks flagged this in the PR workflow.